React-Dojo/react_full_course_2022/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react_full_course_2022",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
```

Comment:
I think testing relating packages should be in `devDepencies`.

Reply:
https://www.youtube.com/watch?v=T_x6QmuJdms

create react app did this :D :D